src/subdataframe/subdataframe.jl

```diff
@@ -51,7 +51,7 @@ immutable SubDataFrame{T <: AbstractVector{Int}} <: AbstractDataFrame
     parent::DataFrame
     rows::T # maps from subdf row indexes to parent row indexes
 
-    function SubDataFrame(parent::DataFrame, rows::T)
+    @compat function SubDataFrame{T}(parent::DataFrame, rows::T) where T
```

Comment:
Yet, shouldn't `T<:AbstractVector{Int}`? Not sure what would happen without it, probably an error a bit later. In his PR Jeff added these (duplicated) constraints.